os/adding-swap.md

```diff
@@ -1,39 +1,16 @@
-# Adding swap in CoreOS/
+# Adding swap in CoreOS
+
 Swap is the process of moving pages of memory to a designated part of the hard disk, freeing up space when needed. Swap can be used to alleviate problems with low-memory environments.
 
 By default CoreOS does not include a partition for swap, however one can configure their system to have swap.
 
-## Manually adding a swapfile on ext4
-
-Run these commands **as root** to create a 512 megabyte swapfile.
-
-```
-$ mkdir /var/vm/
-$ fallocate -l 1024m /var/vm/1GiB.swap
-$ chmod 600 /var/vm/1GiB.swap
-$ mkswap /var/vm/1GiB.swap
-$ swapon /var/vm/1GiB.swap
-```
+## Managing swap with systemd
 
-**Note**: The partition on which you are creating the swapfile must meet the following requirements:
+Systemd can be used to create, begin, and stop a swapfile on your system. Write the following systemd unit file and enable + start the service to add swap to your instance of CoreOS.
```

Comment:
systemd doesn't actually operate on the swapfile; it just operates on the service which manages the swapfile.